core/unit_test/tstHalo.hpp

```diff
@@ -73,6 +73,27 @@ void test1( const bool use_topology )
     EXPECT_EQ( halo->numLocal(), num_local );
     EXPECT_EQ( halo->numGhost(), my_size );
 
+    // Check the owning ranks of the ghost data.
+    auto ghost_own_ranks = halo->ghostOwningRanks();
+    EXPECT_EQ( halo->numGhost(), ghost_own_ranks.size() );
+    auto own_ranks_host = Kokkos::create_mirror_view_and_copy(
+        Kokkos::HostSpace(), ghost_own_ranks );
+    for ( int i = 0; i < my_size; ++i )
```

Comment:
`my_size == halo->numGhost()`?

Reply:
Yes - will update the test to use this value.